shared/common/update-diff.nix

```
@@ -12,6 +12,7 @@
     };
     text = lib.mkOption {
       type = lib.types.str;
+      description = "diff script snippet that runs pre-switch via {option}`system.preSwitchChecks`";
```

```suggestion
      description = "diff script snippet";
```
This option applies to darwin and nixos but pre-switch check is nixos only, it uses an activation script on darwin.